script/go.mod

```diff
@@ -1,12 +1,14 @@
-module exemple/exemple
+module github.com/Eclion/pulumi-playground/example
 
-go 1.20
+go 1.21.0
 
 require (
-	github.com/Eclion/pulumi-playground v0.0.0-20230929090446-15f1174275f5
+	github.com/Eclion/pulumi-playground/dummy-sdk v0.0.0-20230929090446-15f1174275f5
 	github.com/pulumi/pulumi/sdk/v3 v3.86.0
 )
 
+replace github.com/Eclion/pulumi-playground/dummy-sdk => ../dummy-sdk
```

Comment:
This allows us to test against the current SDK, instead of needing to commit and then test.

Reply:
Thanks for the update! I wasn't sure I could use local files, so now I know :)